### ISSUE TITLE
Internal Server Error

### ISSUE THREAD

**Issue by qrlk** [qrlk/adwhore.net#12]
### Internal Server Error fix
**If you have uninstalled extension only because of this error, you don't have to do this. Read this post - it will fix your problem.** 
**Если вы удалили расширение из-за этой ошибки, этот пост должен решить вашу проблему.** 

![image](https://user-images.githubusercontent.com/40423143/90827511-071fe200-e345-11ea-987a-0102eda22f48.png)

**The problem is that ADN failed in registering you as a user for some reason.**
I fixed that in #14. Firefox accepted new version, but it will take 3-4 days for Chrome Web Store to review changes.

**Проблема в том, что ADN не смог выдать вам токен для участия во время установки.**
Эта ошибка была исправлена в #14, Firefox уже принял изменения, Хром стору потребуется 3-4 дня.

Follow this:
![image](https://user-images.githubusercontent.com/40423143/90829659-d6da4280-e348-11ea-92e3-f81d9ab62d6e.png)
![image](https://user-images.githubusercontent.com/40423143/90829705-eb1e3f80-e348-11ea-8f95-3d51f6d28c77.png)

**ADN can be installed from [Firefox Add-ons](https://addons.mozilla.org/ru/firefox/addon/adwhore-net/) and [Chrome Web Store](https://chrome.google.com/webstore/detail/adwhorenet/emfkjghgdkajicmnicojahgojkemagcm).**
Ссылки для скачивания.

**Comment by qrlk** [comment]
**Chrome Web Store has accepted the changes.**  
ADN should update automatically in all major browsers. No further action is required.
I see a lot of registrations outside the installation of the extension, which means the problem has been fixed.

There was also a server side issue that could lead to a 500 error. 
If your IP was in the country database, but it was not associated with a specific country, an error occurred that made it impossible to participate in data collection.  
I linked such cases to an 'unknown' country.

I have enabled server error logging to fix these errors faster.  
I'll keep this issue open for 1-2 days.
